src/System.Windows.Forms/src/System/Windows/Forms/TextBoxBase.TextBoxBaseAccessibleObject.cs

```diff
@@ -127,5 +128,9 @@ internal override SupportedTextSelection SupportedTextSelectionInternal
 
         internal override ITextRangeProvider? GetRangeFromAnnotation(IRawElementProviderSimple annotationElement)
             => _textProvider?.RangeFromAnnotation(annotationElement);
+
+        public override string? KeyboardShortcut => this.TryGetOwnerAs(out TextBoxBase? owner)
+            ? ButtonBaseAccessibleObject.GetKeyboardShortcut(control: owner, useMnemonic: false, previousLabel: PreviousLabel)
```

Comment:
We usually specify parameter names only for literal values (`false`, `"text"`, `10`, `null`, etc.). For variables or properties we can omit it.
```suggestion
            ? ButtonBaseAccessibleObject.GetKeyboardShortcut(control: owner, useMnemonic: false, PreviousLabel)
```
Same for a few other changes in this PR.